parquet/README.md

```diff
@@ -0,0 +1,33 @@
+# Parquet Sample data
+
+This folder contains sample files in various format
+
+## Table of contents
+
+- `alltypes_dictionary.parquet`: Parquet file using int32, int 64, int 96 timestamps
+- `alltypes_plain.parquet`: Parquet file using int32, int 64, int 96 timestamps
+- `alltypes_plain.snappy.parquet`: Parquet file using int32, int 64, int 96 timestamps with snappy compression
+- `customer.impala.GZIP.parquet`: Parquet file with GZIP compression
+- `customer.impala.NONE.parquet`: Parquet file with no compression
+- `customer.impala.SNAPPY.parquet`: Parquet file with snappy compression
+- `flow.snappy.parquet`: Parquet file doubles and with snappy compression
+- `monthlyProductSales.parquet`: Parquet file with whitespace
+- `nation.dict-malformed.parquet`: Non-Standard Parquet file
+- `nation.impala.SNAPPY.parquet`: Parquet file with int32 and binary, snappy compression
+- `nation.impala.GZIP.parquet`: Parquet file with int32 and binary, GZIP compression
+- `nation.impala.NONE.parquet`: Parquet file with int32 and binary, no compression
+- `nested_lists.snappy.parquet`: Nested Parquet file with snappy compression
+- `nested_maps.snappy.parquet`: Nested Parquet map file with snappy compression
+- `nonnullable.impala.parquet`: Nested Parquet file with maps and no nulls
+- `nullable.impala.parquet`: Nested Parquet file with maps and nulls
+- `nulls.snappy.parquet`: Nested Parquet file with maps and nulls, snappy compression
+- `repeated_no_annotation.parquet`: Nested Parquet file with repeat values
+- `stock_simulated.parquet`: Sample stock data in Parquet format
+- `taxi.parquet`: Parquet data with millis timestamps
+- `test_datapage_v2.snappy.parquet`: Nested Parquet list file with snappy compression
+
+
+
+# Source and License
+
+This data was built from data sets publicly available.  It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.
```

Comment:
Do we need/want to provide source URLs like `README.md` does for other data sets?